### ISSUE TITLE
Matplotlib importerror with Matplotlib 3.9.0

### ISSUE THREAD

**Issue by helgelangen** [andersle/gpxplotter#45]
Hello, I'm just getting started with gpxplotter and got the following error when trying to run one of the examples:
```
File "/lib/python3.11/site-packages/gpxplotter/__init__.py", line 40, in <module>
    from .mplplotting import plot_line, plot_filled
  File "/lib/python3.11/site-packages/gpxplotter/mplplotting.py", line 11, in <module>
    from matplotlib.cm import get_cmap
ImportError: cannot import name 'get_cmap' from 'matplotlib.cm' (/lib/python3.11/site-packages/matplotlib/cm.py)
```

Googling the error reveals that several packages using matplotlib have started experiencing this error after matplotlib 3.9.0 came out a few weeks ago. Downgrading matplotlib to the previous version (3.8.4) got rid of the error. Not sure if there's any way for gpxplotter to avoid this problem, other than advising users not to install matplotlib 3.9.0.

**Comment by andersle** [comment]
Thanks for raising this issue! This was addressed by #44, but the PyPI package hadn't been updated yet. The fix should be available now in version [v.0.2.12](https://github.com/andersle/gpxplotter/releases/tag/v0.2.12).

**Comment by helgelangen** [comment]
great, thanks for replying! I had a suspicion it was being worked on as I couldn't find the line causing the error when browsing the github source :)
